network-api/networkapi/settings.py

```diff
@@ -444,7 +465,7 @@
 # Rest Framework Settings
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly',
+        'rest_framework.permissions.AllowAny',
```

Comment:
this feels... dangerous. We don't want any random connection to have [full permissions without authenticating](https://www.django-rest-framework.org/api-guide/permissions/#allowany) @TheoChevalier?

Reply:
That’s a good point… I’m really not familiar with this, but glancing at the doc it seems to be possible to set custom permissions?
I’ve added it as a task to the ticket